src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -114,9 +114,9 @@ function readyFeedbackEditPage() {
 
 function prepareDescription(form) {
     var questionNum = getQuestionNum(form);
-    tinyMCE.get('questiondescription-' + questionNum).save();
-    var descr = form.find('input[name^="questiondescription"]');
-    descr.attr('name', 'questiondescription');
+    var content = tinyMCE.get('questiondescription-' + questionNum).getContent();
+    form.find('input[name=questiondescription]').val(content);
+    form.find('input[name=questiondescription-' + questionNum + ']').prop('disabled', true);
```

Comment:
Just wondering, instead of adding a new `input` field, will replacing the old field `questiondescription-1` with `questiondescription` works?